lib/lockbox/model.rb

```diff
@@ -192,10 +192,11 @@ def update_columns(attributes)
                 # same logic as Active Record
                 # (although this happens before saving)
                 attributes_to_set.each do |k, v|
-                  if respond_to?(:write_attribute_without_type_cast, true)
```

Comment:
Let's keep the current logic to minimize any chance of breakage and add the new logic at the end.

Reply:
Sounds good. I'll take a look at the code and will modify the logic so it finishes with the new logic at the end.